helm/qontract-reconcile/templates/template.yaml

```diff
@@ -521,6 +523,8 @@ objects:
           emptyDir: {}
         - name: fluentd-config
           emptyDir: {}
+        - name: tmp-dir
+          emptyDir: { }
```

Comment:
how about adding a sizeLimit? Just in case

Reply:
we can expand `resources` section above to let each integration config `ephemeral-storage`, https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#setting-requests-and-limits-for-local-ephemeral-storage

Reply:
Right now it's unrestricted as well. Why not add it when it gets a problem.

Reply:
I think that can be done in a new change as we usually don't set it even using emptyDir in many places.

Reply:
why keeping something that we know it's wrong... 😂 

I agree it's much better to write in emptyDir than in the overlay, so this change is good. Since we have the possibility of restricting the size of the emptyDir, it is good to restrict it too in case something gets crazy.